wire-ios-data-model/Tests/Source/ManagedObjectContext/DatabaseMigrationTests+UserClientUniqueness.swift

```diff
@@ -20,14 +20,24 @@ import XCTest
 import Foundation
 @testable import WireDataModel
 
-final class DatabaseMigrationTests_UserClientUniqueness: DatabaseBaseTest {
+final class DatabaseMigrationTests_UserClientUniqueness: XCTestCase {
```

Comment:
Unit tests should be easy to move around and not inherit functions etc. from base classes. "Composition over inheritance".